frontend/src/lib/components/PropertyFilters/Filter/TabbedPropertyFilter.tsx

```diff
@@ -1,16 +1,32 @@
 import React, { useState } from 'react'
 import { Col, Row, Select, Tabs } from 'antd'
 import { keyMapping } from 'lib/components/PropertyKeyInfo'
-import { cohortsModel } from '../../../models/cohortsModel'
+import { cohortsModel } from '../../../../models/cohortsModel'
 import { useValues, useActions } from 'kea'
-import { SelectGradientOverflow } from 'lib/components/SelectGradientOverflow'
-import { Link } from '../Link'
-import { PropertySelect } from './PropertySelect'
+import { SelectGradientOverflow, SelectGradientOverflowProps } from 'lib/components/SelectGradientOverflow'
+import { Link } from '../../Link'
+import { PropertySelect } from '../PropertySelect'
 import { OperatorValueSelect } from 'lib/components/PropertyFilters/OperatorValueSelect'
 import { isOperatorMulti, isOperatorRegex } from 'lib/utils'
+import { propertyFilterLogic } from '../propertyFilterLogic'
+import { PropertyOptionGroup } from '../PropertySelect'
+import { PropertyOperator, PropertyDefinition } from '~/types'
 
 const { TabPane } = Tabs
 
+interface PropertyPaneProps {
+    onComplete: CallableFunction
+    setThisFilter: any
```

Comment:
Why `any`? If it's temporary and needs to be better typed later, can you add a `// TODO:` note?

Reply:
Will add a note. Spent lots of time trying to figure out the appropriate type here and still haven't been able to.

Reply:
Totally fine keeping this as `any` and we can figure out later, some of these require finishing migrating all the rest of the components to TS, so just the reminder note is fine. Don't spend too much time on this.